### ISSUE TITLE
longer commit message when accepting a PR/patch

### ISSUE THREAD

**Issue by gkatsev** [contrib/contrib#26]
It would be nice if we could make longer commit messages when accepting PRs/patches.
For example, allowing for a 50-character first line plus as much text as you want in the rest of the commit message.

**Comment by gkatsev** [comment]
Partially an issue with contrib and partially an issue with the respective `contrib.json`: https://github.com/videojs/video.js/blob/stable/contrib.json#L332

**Comment by heff** [comment]
Yeah, I guess we could have a second prompt for the body of the commit.
It'd also be nice if the message text defaulted to the PR title.
